set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
I'd actually like to see us start updating the versions again. It would be nice not to have to recompile all of the Clojure libraries every time I want to test an older version of one of the services on my machine.